src/CSharp/EasyMicroservices.Serialization.MemoryPack/EasyMicroservices.Serialization.MemoryPack.csproj

```diff
@@ -1,7 +1,19 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
+  <Choose>
+    <When Condition="$(SolutionFileName.Contains('Net7'))">
```

Comment:
the condition are not use, plase add other targetFramwork in section `<Otherwise>`